controllers/contactUsController.js

```diff
@@ -0,0 +1,69 @@
+/* ==================== Start:: imports =================== */ 
+import db from '../../connection/connection-babel/connection';
+import contactUs from '../../models/models-babel/ContactUs.js';
+import Joi from 'joi';
+/* ==================== End:: imports =================== */ 
+
+/* ===== Start:: Validation ===== */ 
+const contactValidation = (data) => {
```

Comment:
Create a folder named `Validation` and move in all these joi validations functions and import it everywhere you are using them